alembic/versions/24533471e302_add_the_verbose_column.py

```diff
@@ -0,0 +1,23 @@
+"""Add the verbose column.
+
+Revision ID: 24533471e302
+Revises: 59ee93c4bf71
+Create Date: 2015-03-20 14:49:57.753166
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '24533471e302'
+down_revision = '59ee93c4bf71'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('preferences',
+                  sa.Column('verbose', sa.Boolean(), server_default="TRUE"))
```

Comment:
No I mean, adding a column with a default value to a table that already has entries (whose value for this column is therefore currently ``null``).
I know I have been bitten by this before and ended up having to:
- create the column
- update all recording in the table for this column (to the default value)
- alter the column to add the default value constraint.

But it could be that alembic takes care of it all already :)